sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -1139,6 +1185,8 @@ def test_lasso_non_float_y(model):
     assert_array_equal(clf.coef_, clf_float.coef_)
 
 
+# FIXME: 'normalize' to be removed in 0.26
```

Comment:
The 0.26 should be renamed to 1.2.

Reply:
This test should not be removed in 1.2 BTW, only the `normalize=True` case. The comment needs an update.